.github/workflows/label-stats.yml

```
@@ -0,0 +1,62 @@
+name: Generate label stats
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  stats:
+    name: Generate label stats
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v3
        with:
          ref: master
```

Make sure we are checking out the master branch here.